document/core/binary/modules.rst

```diff
@@ -148,7 +149,10 @@ Import Section
 The *import section* has the id 2.
 It decodes into the list of :ref:`imports <syntax-import>` of a :ref:`module <syntax-module>`.
 
-$${grammar: {Bimportsec Bimport}}
+$${grammar: {Bimportsec Bimports Bimport}}
+
+.. note::
+   The encoding of :ref:`imports <binary-imports>` beginning with ${:0x01 0xFF} allows multiple imports to be expressed without repeating the module name. ${:0x01 0xFF} decodes as an invalid :ref:`name <binary-name>` for compatibility with older implementations.
```

Comment:
Uh, I don't think this overlay is a good idea. `0x01 0xFF` is a perfectly fine byte vector, it just isn't valid UTF-8. Some systems may not do UTF-8 validation, and even for the standard, we may want to keep it optional as a future option. So I don't think we should go down that particular road.

Instead, I would suggest a new section ID for compact import sections.